code/src/pages/NotFound.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable react/no-unescaped-entities */
+import React from 'react'
+
+export const NotFound = () => {
+  return (
+    <div id="not-found-header"> WRONG WAY, <br /> GO BACK!
+      <p id="not-found-smaller">This page doesn't exist.</p>
+      <p id="not-found"><a href="/" tabIndex="0">Take me back home.</a></p>
```

Comment:
Instead of `<a />` element, you could use `<Link to="/" />` over here

Reply:
changed this anchor tag to <Link>